### DISCUSSION TITLE
Large score differences (>25k) even with <1% difference in accuracy

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#25225] [Q&A]
[as reported by fieryrage on discord](https://discord.com/channels/188630481301012481/384665630319706112/1166841970174464020)

![osu_2023-10-25_13-52-55](https://github.com/ppy/osu/assets/36758269/047b4bbf-cfa7-4a83-b105-f30d41c920ed)

compare scores https://lazer.ppy.sh/scores/2457118023 (leftmost, by Haadez) and https://lazer.ppy.sh/scores/2456985146 (rightmost, by fieryrage) and notice the 0.71% accuracy difference causing a 27.6k score difference

posting discussion to facilitate assessment on whether this is an issue worth addressing

**Comment by peppy** [comment]
What is the expectation here?

**Reply by Zyfarok** [reply]
I don't see this as an issue and the people on the lazer feedback server don't seem to see this as an issue neither.
1% of 1million is 10k, so 27k might seem a lot at first glance for "only" 0.71% acc, but actually this goes in line with the expected behavior: people expect the percents of acc to have more value near 100% accuracy, because it is much more difficult to go from 99% to 100%, than from 80% to 81% for example.
